### PR TITLE
Submit react-react1-week2 homework

### PR DESCRIPTION
**Pull request by Kositthai**

### CODE REVIEW

File: react/react1/week2/todos-app/src/components/AddTodoHandler.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function AppTodoBtn({ todo, setTodo }) {
+  const addTodoHandler = () => {
+    const newTodo = { id: Math.random() * 1000, description: "randomText" };
```
Comment (Divyajg) [inline react/react1/week2/todos-app/src/components/AddTodoHandler.jsx:3]: Instead of using random for ids, research about the unique ids!

**Review by Divyajg — APPROVED** [review]
Good job with the homework.
Make sure a few times giving descriptive names like `renderTodos `instead of `mapTodo`.

File: react/react1/week2/todos-app/src/components/AddTodoHandler.jsx (hunk 1 above)
Comment (Kositthai) [reply react/react1/week2/todos-app/src/components/AddTodoHandler.jsx]: Hi @Divyajg  

Thank you so much for your review! 
I have fixed my random ids and used nanoId instead now and als rename my mapTodo to renderTodos 

Pls let me know if I have to change something else or I can merge it? :)

**Review by Divyajg — APPROVED** [review]
Go ahead 👍

**Comment by Kositthai** [comment]
> Go ahead 👍

Thank you so much :)

**Pull request merged**